### PR TITLE
Sv enchilada article fix

### PR DESCRIPTION
**Pull request by uplne**
This PR removes some margins around enchilada format on content pages. It also reduces size on tablet  breakpoint. Simple but beautiful.

screen shot 2016-03-01 at 16 48 12
screen shot 2016-03-01 at 16 48 23

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/gustyle/gustyle.js [hunk 1]
```diff
@@ -33,12 +35,19 @@ define([
...
+            this.$slot.addClass(classList);
             this.$slot.prepend($.create(template(labelTpl, { data: merge(templateOptions) })));
 
             toggles = new Toggles(this.$slot[0]);
             toggles.init();
         }.bind(this));
     };
+
+    Gustyle.prototype.isContentPage = function () {
+        return !!config.page.isContent;
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/gustyle/gustyle.js]: Do we really want a level of indirection for that?

Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/gustyle/gustyle.js]: Generally speaking I do think a viewmodel should act as a proxy for everything the view depends upon. So I can see advantages as well as disadvantages.

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/gustyle/gustyle.js]: Yeah but I guess a property then would be more appropriate... somehow like you did for the commercial-features refactoring.

**Comment by regiskuckaertz** [comment]
:+1: once you wage in on the property thing

**Pull request merged**